pkgs/applications/graphics/meshlab/default.nix

```diff
@@ -17,13 +19,13 @@
 
 mkDerivation rec {
   pname = "meshlab";
-  version = "2020.03";
+  version = "2020.07";
 
   src = fetchFromGitHub {
     owner = "cnr-isti-vclab";
     repo = "meshlab";
-    rev = "f3568e75c9aed6da8bb105a1c8ac7ebbe00e4536";
-    sha256 = "17g9icgy1w67afxiljzxk94dyhj4f336gjxn0bhppd58xfqh8w4g";
+    rev = "b56ae6f57496b9a42ae1c89638717922ba4a55e3";
```

Comment:
I think you can use `"Meshlab-${version}"` here - they seem to tag their releases with this scheme.

The difference is that when someone sees a bump to an arbitrary commit, they have to (or _should_) go and check it's actually a genuine, mainline revision.

It also makes the job quicker for the next person to do a bump.

Reply:
It's a good idea. Done!